05_iigo/index.tex

```diff
@@ -347,52 +348,69 @@ \section{Accountability Cycle}
+\begin{itemize}
+    \item Each role has the \emph{power} to call an election vote and declare the winner (see Figure~ \ref{fig:cycles_in_IIGO} for the power transfer cycle).
+\end{itemize}
+
+We note that:
 \begin{enumerate}
-    \item The Speaker conducts a vote for the election of the new Judge.
-    \item The Judge conducts a vote for the election of the new President.
-    \item The President conducts a vote for the election of the new Speaker.
+    \item The Speaker conducts an election to appoint a new Judge.
+    \item The Judge conducts an election to appoint a new President.
+    \item The President conducts an election to appoint a new Speaker.
 \end{enumerate}
 Refer to the Figure~ \ref{fig:cycles_in_IIGO} for further clarification about the transfer-of-power cycle.
 
-\begin{rule_IIGO}
-   The role $X$ has the \emph{obligation} to conduct a vote for the election of $Y$ at the end of the term (see Definition~\ref{def:term}) if and only if the role $X$ is accountable to the role $Y$. 
-\end{rule_IIGO}.
+We introduce a \emph{term} length to increase the diversity of the decision group. If the Rule~\ref{rule:roles_must_hold_election} is in play, the roles are obliged to hold an election every N turns. To reduce the scope of the coursework, the term length is defined as a configuration parameter. Thus we reduce the complexity of reasoning the agents have to do with reagard to the transfer-of-power of roles.
+
+\begin{rule_IIGO} \label{rule:roles_must_hold_election}
+    The role $X$ has the \emph{obligation} to conduct a vote for the election of $Y$ if and only if $Y$ has been in power for more turns than the turn length or if role $Z$ has made a monitoring announcement that indicates wrongdoing by $Y$
+\end{rule_IIGO}
+
+\begin{rule_IIGO} \label{rule:must_appoint_elected_island}
+    The role $X$ has the \emph{permission} to \emph{declare the winner} $W$ for an election $E$ if $W = W^{*}$, where $W^{*}$ is the winner produced by \emph{calling} a vote for the election $E$.
+\end {rule_IIGO}
+
+Unlike the speaker, the process of election is more regimented. The power of calling a vote for an election and the power of declaring the result are combined into one action. The decision came as a result of motion to simplify the system for implementation. However, the powers are still, somewhat, kept separate. When facilitating an election the roles still have the option to declare a winner of their choosing. Rule~\ref{rule:must_appoint_elected_island} is what governs this choice. The agents don't, however, have the option to not declare a result at all, holding an election will always result in an declaration of the winner.
```

Comment:
Not sure about the first bit but it should be something like  'Unlike the Speaker's blah blah,...' whatever blah blah is
```suggestion
Unlike the rule vote held by the Speaker, the process of election is more regimented. The power of calling a vote for an election and the power of declaring the result are combined into one action. This decision came as a result of a motion to simplify the system for implementation. However, the powers are still, kept somewhat separate. When facilitating an election, the roles still have the option to declare a winner of their choosing. Rule~\ref{rule:must_appoint_elected_island} is what governs this choice. However, the agents do not have the option to not declare a result at all: holding an election will always result in a declaration of the winner.
```

Reply:
I think @preetl  suggestion is a good one